api/v1alpha1/virtualmachine_types.go

```diff
@@ -132,6 +132,12 @@ type TCPSocketAction struct {
 	Host string `json:"host,omitempty"`
 }
 
+// VirtualMachineDisk defines the properties of a disk.
+type VirtualMachineDisk struct {
+	// Size specifies the disk size in GB
+	Size int `json:"size"`
```

Comment:
The size type should confirm to k8s convention https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md

Reply:
Will fix. Thanks